cmake/sasMacros.cmake

```diff
@@ -79,21 +81,29 @@ macro(add_sas_test_new)
    endif(NOT ARG_CHECKERS AND NOT ARG_MODERNIZE)
 
    set(SAS_CMD "${CMAKE_BINARY_DIR}/scripts/compile")
+   set(FULL_FILENAME "${CMAKE_CURRENT_SOURCE_DIR}/${ARG_FILE}")
+   if(ARG_SA_CONFIGURATION_FILE)
+      set(CONFIGURATION_FILENAME "${CMAKE_CURRENT_SOURCE_DIR}/${ARG_SA_CONFIGURATION_FILE}")
+   endif()
+   if(ARG_WORKING_DIR)
+      set(WORKING_DIR "${CMAKE_SOURCE_DIR}/${ARG_WORKING_DIR}")
+   else()
+      set(WORKING_DIR "${CMAKE_CURRENT_LIST_DIR}")
+   endif()
 
-   set(FULL_FILENAME ${ARG_FILE})
-   # set(FULL_FILENAME ${CMAKE_CURRENT_SOURCE_DIR}/${ARG_FILE})
-   string(REPLACE "/" "." TEST_NAME ${FULL_FILENAME})
+   string(REPLACE "${CMAKE_SOURCE_DIR}/test/" "" TEST_NAME ${FULL_FILENAME})
```

Comment:
Add a comment that this just prepares the test name from the file name of the test source file.